common.py

```diff
@@ -9,262 +9,301 @@
-            if id not in member_roles:
-              nope += 1
-          if nope == 3:
-            await member.remove_roles(targetguild.get_role(703307566654160969))
-
-          embed = discord.Embed(description=f"Role Removed: Removed {role.mention} from {owner.mention}\n**Reason:** No longer Owner of a Server")
-          embed.set_author(name=f"{str(owner)} (ID {owner.id})", icon_url=owner.avatar_url)
-          embed.set_thumbnail(url=owner.avatar_url)
-
-          await self.bot.get_channel(696347411966066689).send(embed=embed)
-      else:
-        if member.id in owner_id_list:
-          await member.add_roles(role)
-
-          embed = discord.Embed(description=f"Role Added: Gave {role.mention} to {owner.mention}\n**Reason:** Owner of a Server but not added by on_guild_join")
-          embed.set_author(name=f"{str(owner)} (ID {owner.id})", icon_url=owner.avatar_url)
-          embed.set_thumbnail(url=owner.avatar_url)
-          
-          await self.bot.get_channel(696347411966066689).send(embed=embed)
-      
-      member_roles = [I_have_role.id for I_have_role in member.roles]
-      for a_id in (738009431052386304, 738009620601241651, 738009624443224195):
-        if a_id in member_roles and 703307566654160969 not in member_roles:
-           await member.add_roles(targetguild.get_role(703307566654160969))
-           await self.bot.get_channel(696347411966066689).send(f"Added highlighted owner of a server to {member.mention}")
-           break
-    await ctx.send("Done!")  
-      
-      
-  @commands.command()
-  @commands.check(is_trusted)
-  async def lookupinfo(self, ctx, mode, *, lookingupwith):
-    if mode == "id":
-      gotguild = self.bot.get_guild(int(lookingupwith))
-    if mode == "name":
-      for guild in self.bot.guilds:
-        if lookingupwith in guild.name:
-          gotguild = guild
-    
-    await ctx.send(cleandoc(f"""
+                           )
+
+        embed = discord.Embed(title=f"{self.bot.user.name} Stats", description=message,
+                              url="https://discord.gg/zWPWwQC", color=0x5bc0ec)
+        embed.set_thumbnail(url="https://publicdomainvectors.org/photos/1462438735.png")
+        await ctx.send(embed=embed)
+
+    @commands.command()
+    @commands.check(is_trusted)
+    async def getinvite(self, ctx, guild: int):
+        invite = None
+        guild = self.bot.get_guild(guild)
+        for channel in guild.channels:
+            try:
+                invite = await channel.create_invite()
+                break
+            except:
+                continue
+        if invite is None:
+            await ctx.send("No channel to make invite for or no perms")
+        else:
+            await ctx.send(str(invite))
+
+    @commands.command()
+    @commands.is_owner()
+    async def sudo(self, ctx, user: typing.Union[discord.Member, discord.User, str], *, message):
+        """mimics another user"""
+        if isinstance(user, str):
```

Comment:
I think this is a better way to do what you're trying to do this is what Pycharm said to do instead